ocw_import/api.py

```diff
@@ -21,6 +20,17 @@
 
 log = logging.getLogger(__name__)
 
+NON_COURSE_IDS = [
```

Comment:
Not sure if this is any better but seems like the name could be more descriptive. If there isn't a good name, a comment explaining why it's needed would be helpful

```suggestion
NON_ID_COURSE_NAMES = [
```